src/bsmHandler.cpp

```diff
@@ -184,6 +184,10 @@ bool BSMHandler::process( const std::string& message_json ) {
     }
 
     metadata["sanitized"] = true;
+    
+    if (metadata.HasMember("asn1")) {
+        metadata["asn1"].SetString("", document.GetAllocator());
```

Comment:
Why not using the remove function `RemoveMember` to remove this member if our intent was to remove this?

Reply:
There wasn't a strong reason either way, but by just removing the data and keeping the field it will keep the same schema as the original message but still sanitize the output.